build/sign.proj

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file something we'd want to share anywhere? I don't see anything `NuGet.Server` specific in here.

Reply:
it does vary per repo. This is essentially a list of tools needed during build